### ISSUE TITLE
🧞 Diagnostic EDMFX Bomex in a box has ~1000 hdf5 files in the artifacts folder

### ISSUE THREAD

**Issue by charleskawczynski** [CliMA/ClimaAtmos.jl#2269]
This job (`🧞 Diagnostic EDMFX Bomex in a box`) has ~1k hdf5 files in the artifact folder, which is probably a diagnostics (frequency) mistake.  Build here: https://buildkite.com/clima/climaatmos-ci/builds/14134#018b5dcc-79bb-481d-bbe5-19bd07aabeb1

@Sbozzolo, maybe this is the issue you were running into?

**Comment by nefrathenrici** [comment]
The diagnostics for this run specifying saving 27 variable values every 600secs, and the total runtime is 21600secs. 
21600 / 600 = 36, so we have 27 variables * 36 timesteps to save diagnostics = 972 files.
This is slightly less than the 1k files, but the old diagnostics are running as well.

Unless I'm missing something major, it looks like this is working as intended?
```
dt_save_to_disk: "10mins"
diagnostics:
  - short_name: ts
    period: 10mins
  - short_name: ta
    period: 10mins
  - short_name: thetaa
    period: 10mins
  - short_name: ha
    period: 10mins
  - short_name: pfull
    period: 10mins
  - short_name: rhoa
    period: 10mins
  - short_name: ua
    period: 10mins
  - short_name: va
    period: 10mins
  - short_name: wa
    period: 10mins
  - short_name: hur
    period: 10mins
  - short_name: hus
    period: 10mins
  - short_name: clw
    period: 10mins
  - short_name: cli
    period: 10mins
  - short_name: hussfc
    period: 10mins
  - short_name: evspsbl
    period: 10mins
  - short_name: arup
    period: 10mins
  - short_name: waup
    period: 10mins
  - short_name: taup
    period: 10mins
  - short_name: thetaaup
    period: 10mins
  - short_name: haup
    period: 10mins
  - short_name: husup
    period: 10mins
  - short_name: hurup
    period: 10mins
  - short_name: clwup
    period: 10mins
  - short_name: cliup
    period: 10mins
  - short_name: waen
    period: 10mins
  - short_name: tke
    period: 10mins
  - short_name: lmix
    period: 10mins
```

**Comment by szy21** [comment]
Yes, I added the configuration and set the output frequency to be the same as the old diagnostics. I think after Gabriele makes the next change the default will be netcdf with one file per variable per reduction, which will significantly reduce the number of files.

**Comment by Sbozzolo** [comment]
Yes, I agree with the previous two comments. Everything is working as expected, and the NetCDF writer will solve the issue (as least, when it comes to the number of files).

> @Sbozzolo, maybe this is the issue you were running into?

No, I switched to working with the NetCDF writer.

**Comment by Sbozzolo** [comment]
Closed with #2179
